test/srt/test_openai_server.py

```diff
@@ -21,38 +24,46 @@ def setUpClass(cls):
     def tearDownClass(cls):
         kill_child_process(cls.process.pid)
 
-    def run_completion(self, echo, logprobs, use_list_input):
+    def run_completion(
+        self, echo, logprobs, use_list_input, parrallel_sample_num, token_input
+    ):
         client = openai.Client(api_key="EMPTY", base_url=self.base_url)
         prompt = "The capital of France is"
+        if token_input:
+            prompt_input = tokenizer.encode(prompt, add_special_tokens=False)
+        else:
+            prompt_input = prompt
 
         if use_list_input:
-            prompt_arg = [prompt, prompt]
+            prompt_arg = [prompt_input, prompt_input]
             num_choices = len(prompt_arg)
         else:
-            prompt_arg = prompt
+            prompt_arg = prompt_input
             num_choices = 1
 
         response = client.completions.create(
             model=self.model,
             prompt=prompt_arg,
-            temperature=0.1,
+            temperature=0,
             max_tokens=32,
             echo=echo,
             logprobs=logprobs,
+            n=parrallel_sample_num,
         )
 
-        assert len(response.choices) == num_choices
+        assert len(response.choices) == num_choices * parrallel_sample_num
 
         if echo:
             text = response.choices[0].text
             assert text.startswith(prompt)
+
         if logprobs:
             assert response.choices[0].logprobs
             assert isinstance(response.choices[0].logprobs.tokens[0], str)
             assert isinstance(response.choices[0].logprobs.top_logprobs[1], dict)
             ret_num_top_logprobs = len(response.choices[0].logprobs.top_logprobs[1])
-            # FIXME: Fix this bug. Sometimes, some top_logprobs are missing in the return value.
-            # assert ret_num_top_logprobs == logprobs, f"{ret_num_top_logprobs} vs {logprobs}"
+            # FIXME: Fix this bug. Sometimes, some top_logprobs are missing in the return value. The reason is that some out_put id maps to the same output token and duplicate in the map
```

Comment:
We can leave it to the next PR. Let us merge this PR first because it fixed some urgent bugs.